fury-on-minikube/README.md

```diff
@@ -46,6 +46,8 @@ cd fury-getting-started/fury-on-minikube
 
     > ⚠️ This command will spin up by default a single-node Kubernetes v1.29.3 cluster, using VirtualBox driver, with 6 CPUs, 16GB RAM and 20 GB Disk.
 
+    > ⚠️ If you are on macOS 13+ then the VirtualBox driver is currently not supported and you should use the Docker or Hyperkit driver.
```

Comment:
Maybe instead of specifying an exception with the Virtualbox driver that may render this statement obsolete in the future (for example if they resolve the issue), we could omit the driver from the above `minikube start` line and remove this statement. @ralgozino what do you think?

Reply:
yes, I would remove the comment and use the default settings. There is no specific reason to use the virtualbox backend as far as I remember.

Reply:
Leaving the default driver makes minikube use docker, if I'm not wrong. On linux there should be no problems, but on platform using Docker Desktop (like Windows and macOS) the services exposed are inaccessible once KFD is up and running therefore it is needed to use `minikube service ingress-nginx --url --namespace ingress-nginx` in order to expose the service ports on the host. Using hyperkit, for example, this additional procedure is not necessary.

When using the `minikube service` procedure to expose services on the hosts makes minikube allocates some random ports accessible from the host loopback address. Because of this we also need to update the ports specified in the guide since it will not be `31443` anymore.

Reply:
Minikube will try all the alternatives if no driver is specified, so if the only available driver is VirtualBox it will use that.

Other than this little remark you're absolutely right, Docker on Mac and Windows will prevent HTTP(S) access to the Docker VM without a tunnel. The `minikube service -n ingress-nginx ingress-nginx --url` is a viable solution, but maybe to keep predictability on the used ports it would be better to use `kubectl port forward`. Maybe we can write something like this at the end of the `Setup Local DNS` section:

> ℹ️ Note
>
> If you are running `minikube` on macOS or Windows, you will need to port-forward the NGINX Ingress ports to localhost to enable access to your exposed applications.
> For example, you can run:
>
>```bash
> kubectl port-forward service/ingress-nginx -n ingress-nginx 31443:443 31080:80
> # Example output:
> Forwarding from 127.0.0.1:31443 -> 8443
> Forwarding from 127.0.0.1:31080 -> 8080
> ```
>
> This command will forward both HTTP and HTTPS ports of the NGINX Ingress Controller to your localhost on ports `31080` and `31442` respectively. Leave that terminal window open while you need access to your applications. Inside your `/etc/hosts` file, instead of the IP address of minikube, you can put `127.0.0.1`.

Let me know what you think!

Reply:
I am with you on this. I think this could be a good solution. I'm going to work on these modifications. I'll push a commit on this PR ASAP.

Reply:
Ok I have removed all the references to the virtualbox driver, updated the `Setup Local DNS` section with the note suggested by @Rick-1990 and fixed all the other issues raised during the review.

Reply:
Love it! I took the liberty to commit a small change to avoid a linting issue when we will build the doc website.
LGTM!